src/reducers/project-text-search.js

```diff
@@ -38,69 +37,68 @@ export type ProjectTextSearchState = {
   status: string
 };
 
-export function initialProjectTextSearchState(): Record<
-  ProjectTextSearchState
-> {
-  return makeRecord(
-    ({
-      query: "",
-      results: I.List(),
-      status: statusType.initial
-    }: ProjectTextSearchState)
-  )();
+export function initialProjectTextSearchState(): ProjectTextSearchState {
```

Comment:
can ProjectTextSearchState be read only `+query` and things like that?

Can we drop `I`. I see it here ` matches: I.List<any>`